.gitattributes

```
@@ -0,0 +1,2 @@
+# Hide our tests from language detection so GitHub correctly detects Topiary as a Rust project
+topiary/tests/** linguist-vendored
```

Good idea. But strictly speaking, I think it should be:

```suggestion
topiary/tests/samples/** linguist-vendored
```